opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/setting-up-minion.adoc

```diff
@@ -1,34 +1,36 @@
 
-== Monitor isolated location with Minion
+== Installing and Configuring a Minion
 
-This section describes how to install the _Minion_ to monitor devices and services in a location which can't be reached from an _{opennms-product-name}_ instance.
+A Minion is an instance of the Karaf OSGi service that enables OpenNMS to monitor devices and services in locations that an OpenNMS instance cannot reach. Minions communicate with these remote devices while OpenNMS performs coordination and task delegation.
 
-=== Objectives
+Minions can operate behind a firewall and/or network address translation (NAT) as long as they can communicate with OpenNMS via ActiveMQ, Apache Kafka, or gRPC. 
```

Comment:
No worries here, just the way where we don't have a name for the Server component. What "Meridian Core" is in our commercial pricing is "OpenNMS Instance" or just "OpenNMS" here or "OpenNMS Horizon" or just "Horizon" in other places. In the Cloud Portal we just call it "Instance".

So the "server" / "main application component" has the following names and means the same thing:

* Meridian
* Meridian Core
* OpenNMS Horizon
* Horizon
* OpenNMS Instance
* Instance
* OpenNMS

Reply:
Agreed. We need to standardize this. Let's decide on what to call it, and then revise all the documentation to be consistent. 
We could start a glossary of terms on Confluence to capture this.